tests/ui/associated-type-bounds/bad-universal-in-dyn-in-where-clause.stderr

```diff
@@ -0,0 +1,8 @@
+error: associated type bounds are only allowed in where clauses and function signatures, not in bound
```

Comment:
Is there a way to make this say "not in bounds"? That sounds more natural to me.